.pre-commit-config.yaml

```
@@ -1,19 +1,21 @@
 repos:
-- repo: https://github.com/asottile/reorder_python_imports
-  rev: v1.9.0
-  hooks:
-  - id: reorder-python-imports
-- repo: https://github.com/psf/black
-  rev: 20.8b1
-  hooks:
-  - id: black
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v2.4.0
-  hooks:
-  - id: end-of-file-fixer
-  - id: check-json
-  - id: check-yaml
-  - id: check-case-conflict
-  - id: check-executables-have-shebangs
-  - id: requirements-txt-fixer
-  - id: flake8
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v1.9.0
+    hooks:
+      - id: reorder-python-imports
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.2.1
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: check-case-conflict
+      - id: check-executables-have-shebangs
+      - id: requirements-txt-fixer
+      - id: flake8
```

Updated version of pre-commit-hooks hooks, and migrated flake8 to its standalone pre-commit-repo as the pre-commit-hooks repo based flake8 hook is deprecated

```suggestion
  - repo: https://gitlab.com/pycqa/flake8
    rev: "3.8.4"
    hooks:
      - id: flake8
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.4.0
    hooks:
      - id: end-of-file-fixer
      - id: check-case-conflict
      - id: check-executables-have-shebangs
      - id: requirements-txt-fixer
```